src/hal/lv_hal_disp.h

```diff
@@ -133,6 +133,9 @@ typedef struct _lv_disp_drv_t {
     /** OPTIONAL: called when driver parameters are updated */
     void (*drv_update_cb)(struct _lv_disp_drv_t * disp_drv);
 
+    /** OPTIONAL: called when start rendering */
+    void (*render_start_cb)(struct _lv_disp_drv_t * disp_drv, struct _lv_disp_t * disp_refr);
```

Comment:
```suggestion
    void (*render_start_cb)(struct _lv_disp_drv_t * disp_drv);
```

`_lv_refr_get_disp_refreshing()` can be used to get the display being rendered.